### PR TITLE
Initial upload

### PR DESCRIPTION
**Pull request by alevsh**

### CODE REVIEW

File: net/release-notes/release-notes-2022/aspose-pdf-for-net-22-1-release-notes/_index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## Improvements and Changes
+
+|**Key**|**Summary**|**Category**|
+| :- | :- | :- |
+|PDFNET-50525|PDF2.0 Add saving document support|New Feature|
+|PDFNET-50523|PDF2.0 Add document loading support|New Feature|
+|PDFNET-50845|Aspose.Pdf.Page.Layers getter throws NullReferenceException when I attempt to access it|Enhancement|
+|PDFNET-44215|Add Popup property for RedactionAnnotation|Enhancement|
+|PDFNET-51073|TextFragmentAbsorber doesn't find text in rectangle by regex|Bug|
+|PDFNET-49025|Copy annotations to new PDF without using XFDF feature|Bug|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2022/aspose-pdf-for-net-22-1-release-notes/_index.md:29]: It seem not a bug, but feature

File: net/release-notes/release-notes-2022/aspose-pdf-for-net-22-1-release-notes/_index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+This page contains release notes information for Aspose.PDF for .NET 22.1.
+
+{{% /alert %}}
+
+## Improvements and Changes
+
+|**Key**|**Summary**|**Category**|
+| :- | :- | :- |
+|PDFNET-50525|PDF2.0 Add saving document support|New Feature|
+|PDFNET-50523|PDF2.0 Add document loading support|New Feature|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2022/aspose-pdf-for-net-22-1-release-notes/_index.md:25]: I think we should uniform these titles:
1) Add document `action` support.
or 
2) PDF 2.0 Document `action` support

File: net/release-notes/release-notes-2022/aspose-pdf-for-net-22-1-release-notes/_index.md (hunk 1 above)
Comment (alevsh) [reply net/release-notes/release-notes-2022/aspose-pdf-for-net-22-1-release-notes/_index.md]: Change to Enhancement

File: net/release-notes/release-notes-2022/aspose-pdf-for-net-22-1-release-notes/_index.md (hunk 2 above)
Comment (alevsh) [reply net/release-notes/release-notes-2022/aspose-pdf-for-net-22-1-release-notes/_index.md:25]: It's just preliminary support of PDF2.0 support (load/save simple files). Full standard support will be available during 2022.

**Pull request merged**